src/EditorFeatures/CSharpTest/Microsoft.CodeAnalysis.CSharp.EditorFeatures.UnitTests.csproj

```diff
@@ -69,6 +65,9 @@
     <PackageReference Include="Microsoft.VisualStudio.Language.CallHierarchy" Version="$(MicrosoftVisualStudioLanguageCallHierarchyVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Language.NavigateTo.Interfaces" Version="$(MicrosoftVisualStudioLanguageNavigateToInterfacesVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Language.StandardClassification" Version="$(MicrosoftVisualStudioLanguageStandardClassificationVersion)" />
+    <!-- Microsoft.VisualStudio.Platform.VSEditor references Microsoft.VisualStudio.Text.Internal since it's needed at runtime; we want to insure we are using
+         it _only_ for runtime dependencies and not anything compile time -->
+    <PackageReference Include="Microsoft.VisualStudio.Text.Internal" Version="$(MicrosoftVisualStudioTextInternalVersion)" IncludeAssets="runtime" />
```

Comment:
💭 @jasonmalinowski How do you feel about using `ExcludeAssets="compile"` and omitting the longer comment? If we couldn't remove the comment from the project file(s), my idea here doesn't seem like a worthwhile change.

Reply:
I guess I'm not sure how that's ay different really. If it wasn't for the Microsoft.VisualStudio.Platform.VSEditor referencing Microsoft.VisualStudio.Text.Internal, then we'd be deleting this line in the first place, but I do want to call out that dependency as being the problem here. If the editor changes that, then somebody should re-evaluate this line.